resources/js/pages/demos/alert/custom-icons.tsx

```
@@ -0,0 +1,30 @@
+import {
+  Alert02Icon,
+  AlertCircleIcon,
+  InformationCircleIcon,
```

Biome (2.4.4)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)